closure/compiler/closure_js_binary.bzl

```diff
@@ -186,9 +186,13 @@ def _impl(ctx):
         args.append("--use_types_for_optimization")
 
     if ctx.attr.output_wrapper:
-        args.append("--output_wrapper=" + ctx.attr.output_wrapper)
+        output_wrapper_file = ctx.actions.declare_file(ctx.attr.name + "_output_wrapper_file")
+        ctx.actions.write(output = output_wrapper_file, content = ctx.attr.output_wrapper)
+        inputs.append(output_wrapper_file)
+        args.append("--output_wrapper_file=" + output_wrapper_file.path)
         if ctx.attr.output_wrapper == "(function(){%output%}).call(this);":
             args.append("--assume_function_wrapper")
```

Comment:
We should probably add an attribute to `closure_js_binary` for this rather than comparing the output wrapper to an exact string, but I consider this out of scope for this PR.

Reply:
Yeah, I found that strange. I don't think a new attribute is necessary though -- you can already pass it in `defs` (which is how we do it for our custom output_wrappers)